ding_campaign.admin.inc

```diff
@@ -144,6 +144,7 @@ function ding_campaign_campaign_rule_form_submit($form, &$form_state) {
       ->condition('cid', $nid)
       ->execute();
 
+    // Clear node cache after campaign rule was deleted from DB.
```

Comment:
I would appreciate a comment describing *why* it is necessary to clear the cache at this point.